src/sql-parser/src/ast/defs/statement.rs

```diff
@@ -2926,16 +2926,15 @@ pub struct RevokeRoleStatement<T: AstInfo> {
     /// The role that is losing a member.
     pub role_name: T::RoleName,
     /// The role that will be removed from `role_name`.
-    pub member_name: T::RoleName,
-    // TODO(jkosh44) Add cascade/restrict.
```

Comment:
I removed cascade/restrict from the scope of this project because it turns out it only applies to grant options which is also out of scope of this project.